framework/encode/vulkan_capture_manager.cpp

```diff
@@ -1800,6 +1825,228 @@ void VulkanCaptureManager::PostProcess_vkEnumeratePhysicalDeviceGroups(
+                                                                     data,
+                                                                     subresource_offsets,
+                                                                     subresource_sizes,
+                                                                     scaling_supported,
+                                                                     true);
+        if (result == VK_SUCCESS)
+        {
+            // Combined size of all layers in a mip level.
+            std::vector<uint64_t> level_sizes;
+
+            uint64_t resource_size = resource_util.GetImageResourceSizesOptimal(image_wrapper->handle,
+                                                                                image_wrapper->format,
+                                                                                image_wrapper->image_type,
+                                                                                image_wrapper->extent,
+                                                                                image_wrapper->mip_levels,
+                                                                                image_wrapper->array_layers,
+                                                                                image_wrapper->tiling,
+                                                                                aspect,
+                                                                                nullptr,
+                                                                                &level_sizes,
+                                                                                true);
+
+            WriteBeginResourceInitCmd(device_wrapper->handle_id, resource_size);
+            GetCommandWriter()->WriteInitImageCmd(api_family_,
+                                                  device_wrapper->handle_id,
+                                                  image_wrapper->handle_id,
+                                                  aspect,
+                                                  image_wrapper->current_layout,
+                                                  image_wrapper->mip_levels,
+                                                  level_sizes,
+                                                  resource_size,
+                                                  data.data());
+            WriteEndResourceInitCmd(device_wrapper->handle_id);
+        }
+    }
+}
+
+void VulkanCaptureManager::PostProcess_vkBindBufferMemory(
+    VkResult result, VkDevice device, VkBuffer buffer, VkDeviceMemory memory, VkDeviceSize memoryOffset)
+{
+    if (result == VK_SUCCESS)
+    {
+        if (IsCaptureModeTrack())
+        {
+            GFXRECON_ASSERT(state_tracker_ != nullptr);
+            state_tracker_->TrackBufferMemoryBinding(device, buffer, memory, memoryOffset);
+        }
+
+        if (IsCaptureModeWrite())
```

Comment:
Is the logic in this PR applicable to AHB and other external memory backing?  Not as part of this PR but that maybe a useful thing to bring up in our AHB conversations.